assets/src/components/higher-order/with-amp-story-settings.js

```diff
@@ -231,6 +231,7 @@ export default createHigherOrderComponent(
 							isSelected={ isSelected }
```

Comment:
There's another issue here -- image block has its own resizable box, however, that does not consider rotation. Any thoughts here? Should we try to hide the original one with CSS and replace it with the custom one instead?

Reply:
We did the same with the block movers already. I think there's no way around doing this for the resizable box as well. Go for it!

Reply:
Just made this change within 3e7c44f.